sphinxlint/checkers.py

```diff
@@ -165,7 +165,10 @@ def check_missing_space_after_role(file, lines, options=None):
     # While this is not:
     #    The :literal:`:exc:`Exceptions``s
     role_body = rf"([^`]|\s`+|\\`|:{rst.SIMPLENAME}:`([^`]|\s`+|\\`)+`)+"
-    suspicious_role = re.compile(f":{rst.SIMPLENAME}:`{role_body}`s")
+    allowed_after_role = " |"  # '|' is allowed for roles in tables.
```

Comment:
Aren't commas and other punctuation allowed too?

Reply:
You're right, I updated the list, does it looks better?